src/client.coffee

```diff
@@ -34,34 +35,55 @@ module.exports = class Client
   constructor: (provider_key, options) ->
```

Comment:
The issue with passing service_token in the initialiser is that the whole instance of the client will be usable only with one service. So it would make no sense to pass service id in the method as only the one paired with the token would work.

This is very similar to https://github.com/3scale/3scale_ws_api_for_php/pull/15#issuecomment-278256001 and https://github.com/3scale/3scale_ws_api_for_ruby/pull/49#discussion_r96654289